db/migrate/20220711110429_create_articles.rb

```diff
@@ -1,11 +0,0 @@
-class CreateArticles < ActiveRecord::Migration[7.0]
-  def change
-    create_table :articles do |t|
-      t.string :author
-      t.string :title
-      t.string :content
-
-      t.timestamps
-    end
-  end
-end
```

Comment:
Be careful when doing this... As I explained in the first day when we setup Rails, changing a migration file does nothing unless it's rollbacked (and that should happen during the development phase or a major issue when deploying). This way, deleting a migration file does nothing -> you'll have the Articles table forever in your Database unless you Drop the full database and start over again (You can already see why this is a problem and should be avoided, but as you haven't anything deployed, we can assume all of you will start fresh again).

But again, be careful with this in the future.

Reply:
Understood.